crates/db-boot/README.md

```diff
@@ -0,0 +1,21 @@
+# db-boot
+
+## Design
+
+The `db-boot` crate is a collection of constants and utilities to help define VM specification in the VM boot stage.
```

Comment:
The `db-boot` crate is a collection of constants, structs and utilities used to boot virtual machines.

Reply:
Changed.